tests/test_attention_ops.py

```diff
@@ -6,14 +6,9 @@
 import pytest
 import torch
 import triton
+import vllm.vllm_flash_attn.flash_attn_interface  # noqa: F401
```

Comment:
Import vllm in the test function and use skipif to check whether vllm is installed. There is no install vllm on CI.